src/Microsoft.Framework.Configuration.Json/JsonConfigurationSource.cs

```diff
@@ -78,7 +80,21 @@ public override void Load()
         internal void Load(Stream stream)
         {
             JsonConfigurationFileParser parser = new JsonConfigurationFileParser();
-            Data = parser.Parse(stream);
+            try
+            {
+                Data = parser.Parse(stream);
+            }
+            catch(JsonReaderException e)
+            {
+                string errorLine = string.Empty;
+                if (File.Exists(Path))
+                {
+                    // Read the JSON file and get the line content where the error occurred.
+                    errorLine = File.ReadLines(Path).Skip(e.LineNumber - 2).FirstOrDefault().Trim();
```

Comment:
We want to read `LineNumber-1` since `File.ReadLines` starts with 0 index. So I am skipping until `LineNumber - 2` to read `LineNumber - 1`